datumaro/plugins/data_formats/synthia/importer.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os.path as osp
+
+from datumaro.components.format_detection import FormatDetectionConfidence, FormatDetectionContext
+from datumaro.components.importer import Importer
+
+from .format import SynthiaAlPath, SynthiaFormatType, SynthiaRandPath, SynthiaSfPath
+
+
+class _SynthiaImporter(Importer):
+    FORMAT = None
+    META_FOLDERS = []
+
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> FormatDetectionConfidence:
+        for folder in cls.META_FOLDERS:
+            if not osp.isdir(osp.join(context.root_path, folder)):
+                context.fail("Any Synthia format is not detected.")
+
+        return FormatDetectionConfidence.MEDIUM
+
+    @classmethod
+    def find_sources(cls, path):
+        return [{"url": path, "format": cls.FORMAT}]
+
+
+class SynthiaRandImporter(_SynthiaImporter):
+    FORMAT = SynthiaFormatType.synthia_rand.name
+    META_FOLDERS = []
+    for prefix in vars(SynthiaRandPath).values():
+        if isinstance(prefix, str) and "datumaro" not in prefix:
+            META_FOLDERS.append(prefix)
```

Comment:
```suggestion
    META_FOLDERS = SynthiaRandPath.meta_folders()
```
I recommend you wrap these lines up to a `classmethod` of `SynthiaRandPath` for better readability.

Reply:
I updated at the [241b5548af138265e96f7c871eec0b522a225448](https://github.com/openvinotoolkit/datumaro/pull/987/commits/241b5548af138265e96f7c871eec0b522a225448)